viewing_party/party.py

```diff
@@ -1,23 +1,187 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+from enum import unique
+from logging.handlers import RotatingFileHandler
+from re import L, U
+import re
+from tests.test_constants import MOVIE_TITLE_1, USER_DATA_2
+import copy
+import statistics
 
+
+def create_movie(movie_title, genre, rating):
+    if not movie_title or not genre or not rating:
+        return None
+    new_movie = {
+        "title" : movie_title, 
+        "genre": genre,
+        "rating": rating }
+    return new_movie
+
+def add_to_watched(user_data, movie):
+     user_data["watched"].append(movie)
+     return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie_to_watch in user_data["watchlist"]:
+        if movie_to_watch["title"] == title:
+            user_data["watchlist"].remove(movie_to_watch)
+            user_data["watched"].append(movie_to_watch)
+    return user_data
+
+def watched_movie(user_data, title):

```

Comment:
I see that you created this helper function but you aren't using it. What were your plans to use it?